plugins/kubernetesapi-report/src/main/java/com/freenow/sauron/plugins/KubernetesApiReport.java

```diff
@@ -62,6 +67,11 @@ public DataSet apply(PluginsConfigurationProperties properties, DataSet input)
                 .map(Map.class::cast)
                 .map(Map.class::cast)
                 .ifPresent(propFilesCheck -> kubernetesPropertiesFilesReader.read(input, serviceLabel, propFilesCheck, apiClient));
+
+            properties.getPluginConfigurationProperty(PLUGIN_ID, PRODUCTION_READINESS_CHECK).filter(Map.class::isInstance)
+                .map(Map.class::cast)
+                .map(Map.class::cast)
+                .ifPresent(prodReadinessCheck -> kubernetesContainersReader.read(input, serviceLabel, apiClient));
```

Comment:
Here we want to pass the property that has to be read from the container in the arguments of `kubernetesContainersReader.read(...`